airflow/models/dagbag.py

```diff
@@ -391,6 +391,10 @@ def collect_dags(
             'dagbag_import_errors', len(self.import_errors), 1)
         self.dagbag_stats = sorted(
             stats, key=lambda x: x.duration, reverse=True)
+        for file_stat in self.dagbag_stats:
+            Stats.timing('dag.loading-duration.{}'.
+                         format(file_stat.dags),
```

Comment:
I think it is possible that we have multiple `DAG` s created in a file and that may break the stats name.

Reply:
`str([dag.dag_id for dag in found_dags])` has taken care of that.

Reply:
but you are right, if we have dynamic DAGs with the same fileloc. It is a bit hard to trace.

Reply:
Uh this is what I mean:
```
>>> str(['dag_id1', 'dag_id2'])
"['dag_id1', 'dag_id2']"
```

Reply:
I update the name to be `dag.loading-duration.dag_id1` if we have one dag per file and `dag.loading-duration.dag_id1|dag_id2` if we have multiple dags per file.